liferay-echo/liferay-portal
# LPS-134358

## What is included in this PR
Allow users to nest a collection display within a collection display

## How to test this:
- Deploy the configuration file attached in https://issues.liferay.com/browse/LPS-134361
- Create a couple of collections and try to nest those
- For the nested collection the item selector to choose a collection should include the related collection providers, which allow to select a related collection for the item of the outer collection being rendered.

## What is out of this PR
Drag and drop will work fine both in page structure and the page editor itself. The only problem is that you cannout move an item from inside the collection to the outside, but this will be tackled in another subtask
